src/shm_main.c

```diff
@@ -902,11 +902,11 @@ sr_shmmain_check_conn_lock(sr_cid_t cid, int *conn_alive)
         return err_info;
     }
     if (fl.l_type == F_UNLCK) {
-        SR_LOG_INF("Lock test found no connection for CID %ld.", cid);
+        SR_LOG_INF("Lock test found no connection for CID %"PRIu32".", cid);
         *conn_alive = 0;
     } else {
         /* we cannot get the lock, it must be held by an alive connection */
-        SR_LOG_INF("Lock test found alive connection by pid [%ld] for CID %ld.", fl.l_pid, cid);
+        SR_LOG_INF("Lock test found alive connection by pid [%ld] for CID %"PRIu32".", fl.l_pid, cid);
```

Comment:
Here should be `(long)fl.l_pid` (that is the most portable way of printing it, AFAIK).

Reply:
Cast added.